src/modules/app/scanner.js

```diff
@@ -4,7 +4,7 @@ import omit from 'lodash/omit';
 
```

Comment:
Name of the file could be scannerStore or we should change other store files name?